qds_sdk/commands.py

```diff
@@ -1347,6 +1347,44 @@ def parse(cls, args):
         v["command_type"] = "DbTapQueryCommand"
         return v
 
+class JupyterNotebookCommand(Command):
+    usage = "jupyternotebookcmd <submit|run> [options]"
+
+    optparser = GentleOptionParser(usage=usage)
+    optparser.add_option("--path", dest="path", help="Path including name of the Jupyter notebook to be run with extension.")
+    optparser.add_option("--cluster-label", dest="label", help="Label of the cluster on which the this command should be run. If this parameter is not specified then label = “default” is used.")
+    optparser.add_option("--arguments", dest="arguments", help="Valid JSON to be sent to the notebook. Specify the parameters in notebooks and pass the parameter value using the JSON format. key is the parameter’s name and value is the parameter’s value. Supported types in parameters are string, integer, float and boolean.")
```

Comment:
please fix the non-ascii characters (use a better IDE/Editor) :)
```suggestion
    optparser.add_option("--cluster-label", dest="label", help="Label of the cluster on which the this command should be run. If this parameter is not specified then label = "default" is used.")
    optparser.add_option("--arguments", dest="arguments", help="Valid JSON to be sent to the notebook. Specify the parameters in notebooks and pass the parameter value using the JSON format. key is the parameter's name and value is the parameter's value. Supported types in parameters are string, integer, float and boolean.")
```